pydantic/import_helper.py

```diff
@@ -0,0 +1,382 @@
+    'pydantic.errors.IPv6InterfaceError': V2MigrationRemovedException('TODO'),
+    'pydantic.errors.ColorError': V2MigrationRemovedException('TODO'),
+    'pydantic.errors.StrictBoolError': V2MigrationRemovedException('TODO'),
+    'pydantic.errors.NotDigitError': V2MigrationRemovedException('TODO'),
+    'pydantic.errors.LuhnValidationError': V2MigrationRemovedException('TODO'),
+    'pydantic.errors.InvalidLengthForBrand': V2MigrationRemovedException('TODO'),
+    'pydantic.errors.InvalidByteSize': V2MigrationRemovedException('TODO'),
+    'pydantic.errors.InvalidByteSizeUnit': V2MigrationRemovedException('TODO'),
+    'pydantic.errors.MissingDiscriminator': V2MigrationRemovedException('TODO'),
+    'pydantic.errors.InvalidDiscriminator': V2MigrationRemovedException('TODO'),
+    # pydantic.fields
+    'pydantic.fields.Required': V2MigrationRemovedException('pydantic.Required has been removed in favour of ...'),
+    # 'pydantic.fields.UndefinedType': V2MigrationRemovedException('TODO'),  # Moved
+    # 'pydantic.fields.Undefined': V2MigrationRemovedException('TODO'),  # Moved
+    'pydantic.fields.SHAPE_SINGLETON': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_LIST': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_SET': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_MAPPING': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_TUPLE': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_TUPLE_ELLIPSIS': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_SEQUENCE': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_FROZENSET': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_ITERABLE': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_GENERIC': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_DEQUE': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_DICT': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_DEFAULTDICT': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_COUNTER': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.SHAPE_NAME_LOOKUP': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.MAPPING_LIKE_SHAPES': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.ModelField': V2MigrationMovedWarning('TODO'),
+    'pydantic.fields.DeferredType': V2MigrationMovedWarning('TODO'),  # Moved
+    'pydantic.fields.is_finalvar_with_default_val': V2MigrationMovedWarning('TODO'),
+    # pydantic.main
+    'pydantic.main.validate_custom_root_type': V2MigrationMovedWarning('TODO'),
+    'pydantic.main.generate_hash_function': V2MigrationMovedWarning('TODO'),
+    'pydantic.main.ANNOTATED_FIELD_UNTOUCHED_TYPES': V2MigrationMovedWarning('TODO'),
+    'pydantic.main.UNTOUCHED_TYPES': V2MigrationMovedWarning('TODO'),
+    'pydantic.main.validate_model': V2MigrationMovedWarning('TODO'),
+    # pydantic.mypy NOTE: ERROR_ORM still exists, should it be renamed to ERROR_ATTRIBUTES
```

Comment:
@samuelcolvin for this, while looking through v1-> v2 I noticed that ERROR_ORM still exists, should it be renamed to ERROR_ATTRIBUTES during the migration ?

Reply:
Still present, just ignored at the moment in the tests, let me know your preference 👍🏼

Reply:
Not really sure - I think we might just have not got there yet.